gatsby-node.js

```diff
@@ -28,13 +52,6 @@ exports.createPages = async ({ graphql, actions }) => {
   const projectPageTemplate = require.resolve('./src/templates/project.js')
   if (projectResults.data) {
     projectResults.data.giveth.projects.forEach(project => {
-      console.log(
-        'theproject ====>',
-        project.title,
-        'theslug===>',
```

Comment:
@geleeroyale thanks for the feedback, I understand your frustration, so thanks for communicating it so clearly.

I'm sorry for my part in this. 

> Mateo got stuck on an error that was introduced in this branch and I have to take time and effort to troubleshoot the netlify build and review a PR that is not expected at this point in time

The build broke after I tried to fix the warnings and clean up the code. The console was FULL of warnings, as I'm sure you know and it's my opinion that the quality of the code should be of a certain standard before the MVP is complete. I accept that I could have done that on a separate branch though.

> Furthermore - white-labeling is a big topic and the rest of the Giveth team also has plans for this, so it would have been nice to discuss the approach before coding something up. Even tho some groundwork exists here, it is far from production ready and I am also not a big supporter of the approach taken.

It's minimum viable white label, the minimal I need for my use case. I could have operated on a fork, but we would have had the same issue trying to get any of my improvements to the giveth code back into the main repo. My timelines didn't allow me to wait and get consensus, but I invested as little as possible in this development, and I am aware that a CMS integrated version of the white label is probably more desirable, but not necessary at this time.

> Some necessary code was just deleted without asking why it was put there. I was lucky to catch it, but now I have to remember to put it back in.
Are you referring to the console.logs above? If so I would argue that there should be a comment there telling devs not to remove it. It looked like cosole.logs that had been forgotten to me, it's a very common mistake to find, so I must be mistaken in my understanding?

Reply:
I would also add that I asked if there was any objections to merging these changes, and there was plenty of opportunity to voice concerns, and we absolutely didn't need to merge the PR, in fact, I'm happy to back these changes out and work on a fork if you prefer that.